pom.xml

```diff
@@ -235,17 +235,17 @@
 			<dependency>
 				<groupId>org.hibernate</groupId>
 				<artifactId>hibernate-core</artifactId>
-				<version>${hibernateVersion}</version>
+				<version>6.0.0.Alpha2</version>
 			</dependency>
 			<dependency>
 				<groupId>org.hibernate</groupId>
 				<artifactId>hibernate-c3p0</artifactId>
-				<version>${hibernateVersion}</version>
+				<version>6.0.0.Alpha2</version>
 			</dependency>
 			<dependency>
 				<groupId>org.hibernate</groupId>
 				<artifactId>hibernate-ehcache</artifactId>
-				<version>${hibernateVersion}</version>
+				<version>5.4.2.Final</version>
```

Comment:
you dont need to change this again once already changed in the properites tag

Reply:
Sure I have done so already and it will reflect in the PR when I push